pkgs/servers/mastodon/default.nix

```diff
@@ -21,6 +22,14 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-Ngfs15YKLfSBOKju3BzpZFnenB370jId2G1g9Qy1y5w=";
   };
 
+  patches = [
+    # Fix indexing statuses in ElasticSearch
+    (fetchpatch {
+      url = "https://github.com/mastodon/mastodon/pull/17912.patch";
```

Comment:
That PR got merged. Could you pull in https://github.com/mastodon/mastodon/commit/ef196c913c77338be5ebb1e02af2f6225f857080 please?